content/all-spaces/disconnected-spaces/helm-reference.md

```diff
@@ -81,6 +83,11 @@ This reference provides detailed documentation on the Upbound Space Helm chart.
 | apollo.serviceAccount.annotations | object | `{}` | None |
 | apollo.serviceAccount.create | bool | `true` | None |
 | apollo.serviceAccount.name | string | `"mxe-apollo"` | None |
+| authentication | object | | Authentication options |
+| authentication.hubIdentites | bool | `false`  | This enables respecting built in Kubernetes identities (client certificate, managed kubernetes OIDC, Kubernetes Groups, etc) specified  within the Connected Space's hub. |
+| authentication. structuredConfig | bool | `false` | None |
```

Comment:
Writes file `auth-config.yaml` into a ConfigMap `spaces-router-auth-config`, which contains a key `config.yaml` with Kubernetes Structured Authentication Configuration.

Reply:
Which attribute does that description belong to?